l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -343,7 +343,13 @@ def setCedentePrestatore(self, company, fatturapa):
     def _setDatiAnagraficiCessionario(self, partner, fatturapa):
         fatturapa.FatturaElettronicaHeader.CessionarioCommittente.\
             DatiAnagrafici = DatiAnagraficiCessionarioType()
-        if not partner.vat and not partner.fiscalcode:
+        if not partner.vat and not partner.fiscalcode and partner.codice_destinatario == 'XXXXXXX':
```

Comment:
https://github.com/OCA/l10n-italy/pull/861#discussion_r249374170

Reply:
Se codice_destinatario='XXXXXXX' allora countrycode<>"IT" per definizione, quindi non serve testare il countrycode

Reply:
What if `partner.country_id` is empty?